patches/css-tree@2.3.1.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/package.json b/package.json
+index 934e699f19c6f1a265c1ef29121c6dde92cd25ed..a54e28a44de8e1adcf775be9564c2ad8b15b5c98 100644
+--- a/package.json
++++ b/package.json
+@@ -18,11 +18,11 @@
+     "validation"
+   ],
+   "type": "module",
+-  "module": "./lib/index.js",
++  "module": "./dist/csstree.esm.js",
```

Comment:
patching css-tree to point to the built version so that packages (eg. the builder) that use `css-data` on the client don't fail:

![Screen Shot 2023-05-22 at 10 51 28 AM](https://github.com/webstudio-is/webstudio-builder/assets/71319175/2d5a8427-1b38-4d99-8978-6151665d5809)